### PR TITLE
Update the docs to reflect the new user scoped alerts

### PR DESCRIPTION
**Pull request by ajaygit158**
@sanjaypujare Kindly review

### CODE REVIEW

**Comment by sanjaypujare** [comment]
@ajaygit158 sorry didn't get to this in time. This needs to be rebased and I will take a look

**Comment by ajaygit158** [comment]
@sanjaypujare Updated the PR including the latest commits. Kindly revert if you see any issue

File: docs/dtgateway_systemalerts.md [hunk 1]
```diff
@@ -140,6 +141,13 @@ user to re-enter the previous password. Note that the JSON sample following the
 The GET API returns the existing SMTP configuration in the gateway. As mentioned above, the SMTP-password
 value is never returned for security reasons.
 
+## Authorization Support for Alerts and Topics
+
+Starting with RTS 3.8, access to alerts will be restricted based on alert owner. A user can view and manage only his own alerts. A user with admin role will have complete access to all alerts including those of other users.
+
+A user will also have restricted access to topics. A user can only view and access topics for applications started by him. Access to topics "cluster.metrics" and "applications" will be restricted to users who have permissions to view system apps (VIEW_SYSTEM_APPS). A user with admin role will have access to all topics of all users as well as cluster.metrics and applications.
```
Comment (sanjaypujare) [inline docs/dtgateway_systemalerts.md]: VIEW_SYSTEM_APPS is being replaced. Can you check with @ShunxinLu  and see https://github.com/DataTorrent/DTX/pull/786 ?

Comment (ShunxinLu) [reply docs/dtgateway_systemalerts.md]: @ajaygit158 VIEW_SYSTEM_APPS permission is removed as part of the requirement of removing reference of app data tracker. We had a discussion and decided that we will replace it with VIEW_GLOBAL_CONFIG in this case.

**Comment by ajaygit158** [comment]
@sanjaypujare I have changed the document to reflect updated permission name. Kindly review.

**Pull request merged**